ui/webui/src/components/storage/InstallationMethod.jsx

```diff
@@ -469,17 +469,33 @@ const ModifyStorageButton = ({ idPrefix, isBootIso, onModifyStorage }) => {
     );
 };
 
-const startBlivet = (onStart, onFinished, errorHandler) => {
+const startBlivet = (onStart, onFinished, errorHandler, blivetProcessRef) => {
     console.log("Spawning Blivet.");
-    cockpit.spawn(["blivet-gui"], { err: "message" })
-            .then(() => {
+    blivetProcessRef.current = cockpit.spawn(["blivet-gui"], { err: "message" })
+            .then((process) => {
                 console.log("Blivet exited.");
+                blivetProcessRef.current = null;
                 return onFinished();
             })
-            .catch(errorHandler);
+            .catch((message) => { blivetProcessRef.current ? errorHandler(message) : console.log("Blivet exited with error", message) });
     onStart();
 };
 
+const stopBlivet = (onClose, onStopped, errorHandler, blivetProcessRef) => {
+    if (blivetProcessRef.current) {
+        console.log("Stopping Blivet");
+        blivetProcessRef.current = null;
+        cockpit.spawn(["killall", "blivet-gui"], { err: "message" })
+                .then(() => {
+                    console.log("Blivet stopped.");
```

Comment:
Maybe "Blivet GUI" there as well ? Could be otherwise easily confused with something going wrong the Blivet library itself.

Reply:
Updated in a separate commit.